deployment/submit_emr_job.py

```diff
@@ -0,0 +1,176 @@
+                            "Classification": "spark-env",
+                            "Properties": {},
+                            "Configurations": [
+                                {
+                                    "Classification": "export",
+                                    "Configurations": [],
+                                    "Properties": {
+                                        "LC_ALL": "en_US.UTF-8",
+                                        "LANG": "en_US.UTF-8",
+                                        "AWS_S3_ACCESS_KEY_ID": config.AWS_S3_ACCESS_KEY_ID,
+                                        "AWS_S3_SECRET_KEY_ID": config.AWS_S3_SECRET_KEY_ID
+                                    }
+                                }
+                            ]
+                        }
+                    ]
+                }
+            ],
+        },
+        BootstrapActions=[
+            {
+                'Name': 'Metadata setup',
+                'ScriptBootstrapAction': {
+                    'Path': s3_bootstrap_uri
+                }
+            }
+        ],
+        Steps=[
+            {
+                'Name': 'Setup Debugging',
+                'ActionOnFailure': 'TERMINATE_CLUSTER',
+                'HadoopJarStep': {
+                    'Jar': 'command-runner.jar',
+                    'Args': ['state-pusher-script']
+                }
+            },
+            {
+                'Name': 'setup - copy files',
+                'ActionOnFailure': 'TERMINATE_CLUSTER',
+                'HadoopJarStep': {
+                    'Jar': 'command-runner.jar',
+                    'Args': ['aws', 's3', 'cp', s3_uri, '/home/hadoop/']
+                }
+            },
+            {
+                 'Name': 'setup - unzip files',
+                 'ActionOnFailure': 'TERMINATE_CLUSTER',
+                 'HadoopJarStep': {
+                     'Jar': 'command-runner.jar',
+                     'Args': ['unzip', '/home/hadoop/' + s3_key, '-d', '/home/hadoop']
+                 }
+            },
+            # {
+                # 'Name': 'Install Tensorflow',
+                # 'ActionOnFailure': 'TERMINATE_CLUSTER',
+                # 'HadoopJarStep': {
+                    # 'Jar': 'command-runner.jar',
+                    # 'Args': ['/bin/bash', '/home/hadoop/tensorflow_install.sh']
+                # }
+            # },
+            {
+                'Name': 'Run training job',
+                'ActionOnFailure': 'TERMINATE_CLUSTER',
+                'HadoopJarStep': {
+                    'Jar': 'command-runner.jar',
+                    'Args': ['python3.6', '/home/hadoop/CVAE/test_vae_package_data.py']
+                }
+            }
+        ],
+        Applications=[{'Name':'MXNet'}],
+        VisibleToAllUsers=True,
+        JobFlowRole='EMR_EC2_DefaultRole',
+        ServiceRole='EMR_DefaultRole'
+    )
+
+    output = {}
+    if response.get('ResponseMetadata').get('HTTPStatusCode') == 200:
```

Comment:
if `response.get('ResponseMetadata')` key doesn't exist it will throw `AttributeError`
```python
response.get('ResponseMetadata', {}).get('HTTPStatusCode')
```